pyidr/study_parser.py

```diff
@@ -405,25 +405,18 @@ def __str__(self):
 
     def generate_description(self, component):
         """Generate the description of the study/experiment/screen"""
-        publication_title = ""
-        if component["Study Publication Title"]:
-            # Only display the first publication
-            publication_title = (
-                "Publication Title\n%(Study Publication Title)s" %
-                component).split('\t')[0] + "\n\n"
         if "Type" in component:
             key = "%s Description" % component["Type"]
         else:
             key = "Study Description"
-        component_title = (
-            "%s\n%s" % (key, component[key]))
+        component_title = component[key]
         if "Study Version History" in component:
             history = ("\n\nVersion History\n%s" %
```

Comment:
If we are looking into removing publication title from the description, a question is whether this heading should be kept at all for the relevant studies or if the value of this field should be more simply appended with 1-2 newlines

Reply:
by "this heading" do you mean "Study Version History"? I'm happy to remove that heading too if it's not needed?

Reply:
yes, I meant `Study Version History` but deferring to you, @khaledk2 and @francesw on what makes the most sense both in terms of UI and searchability.